types/shippo/index.d.ts

```diff
@@ -1,70 +1,135 @@
-// Type definitions for shippo 1.5
+// Type definitions for shippo 1.6
 // Project: https://github.com/goshippo/shippo-node-client
-// Definitions by: Saiichi <https://github.com/saiichihashimoto>
+// Definitions by: Saiichi <https://github.com/saiichihashimoto>, Eric Johnson <https://github.com/ejhnsn>
```

Comment:
Sounds like it would be appreciated if you dropped the current owner.

Reply:
removed